debian/source/lintian-overrides

```diff
@@ -0,0 +1,2 @@
+# Conflict due to Ubuntu version conventions
+cloud-init source: binary-nmu-debian-revision-in-source
```

Comment:
[Reported](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1014584) and referenced.

Reply:
I've commented on the lintian bug as well. I think we can discuss as a team whether to change or diminished version `~18.04.1` vs. augmented version strategy `.18.04.1` even though I feel like this is a lintian versioning bug. Since we are now appending the Ubuntu development version suffix for all releases in kinetic, changing to the augmented `.22.04.1` suffix for jammy would still preserve a package version upgrade path to kinetic's comparable suffix `.22.10.1`.